### DISCUSSION TITLE
使用umi max + ant design pro 创建的项目，生成的页面中看到有报错信息

### DISCUSSION THREAD

**Discussion by liaodalin19903** [umijs/umi#10947] [Q&A]
```
文件名“/Users/marl/Desktop/Test/js/UmiJS/umijs-max-demo-02/node_modules/@ant-design/pro-components/es/index.d.ts”仅在大小写方面与包含的文件名“/users/marl/desktop/test/js/umijs/umijs-max-demo-02/node_modules/@ant-design/pro-components/es/index.d.ts”不同。
  程序包含该文件是因为:
    通过 "/Users/marl/Desktop/Test/js/UmiJS/umijs-max-demo-02/node_modules/@ant-design/pro-components" 从具有 packageId "@ant-design/pro-components/es/index.d.ts@2.4.4" 的文件 "/Users/marl/Desktop/Test/js/UmiJS/umijs-max-demo-02/src/.umi/plugin-layout/types.d.ts" 引用了库类型
    通过 "/Users/marl/Desktop/Test/js/UmiJS/umijs-max-demo-02/node_modules/@ant-design/pro-components" 从具有 packageId "@ant-design/pro-components/es/index.d.ts@2.4.4" 的文件 
```

![image](https://user-images.githubusercontent.com/12727582/231327698-b0640049-db91-4cc8-9d4c-054030419974.png)

**Comment by fz6m** [comment]
没遇到你这个问题，可能和你的 vscode 版本，工作区位置，工作区配置，vscode 缓存，ts 引擎版本等有关，自己排查下。

**Reply by Wei0809** [reply]
你好，能再排查一下这个问题吗，我使用npm安装也遇到了这个问题，并且在使用pnpm后正常了，我看下面的评论中使用yarn也有这个问题，是为了兼容pnpm导致的吗？

**Comment by Sallyfafafa** [comment]
同样遇到这个问题，请问你解决了吗，我用的也是 vscode

**Reply by liaodalin19903** [reply]
还没有解决。

**Reply by Sallyfafafa** [reply]
vscode中的 tsconfig.json 文件我加了这个配置，不会飘红了，但是还是没有智能引入的提示...
```
{
  "compilerOptions": {
    "forceConsistentCasingInFileNames": false,
    }
}

```

**Comment by weicong** [comment]
![image](https://user-images.githubusercontent.com/1157868/232426770-2f5cfcd1-d627-42d6-b443-888a05a8a712.png)

~~估计只有 Windows 才会报错~~

**Reply by xieliangzhen** [reply]
mac会有类似报错，我是window正常

**Reply by weicong** [reply]
我用 yarn 有这个问题，改用 pnpm 就正常了。

**Comment by Wei0809** [comment]
同样遇到了这个问题。。。以前的项目就没有，把package.json修改成以前的版本也没法解决

**Comment by fz6m** [comment]
在 #11015 中修复。

**Comment by cinema8150** [comment]
试试这个呢：https://github.com/ant-design/pro-components/issues/6592

**Comment by fz6m** [comment]
如果还有类似的类型报错问题的，首先建议尝试把 vscode 升级到最新版本、umi 的依赖版本升级到最新，若未显示安装 ant procomponents 则显示安装到 `package.json` 。
